src/Nest/CommonAbstractions/Request/RequestBase.cs

```diff
@@ -47,12 +47,29 @@ protected RequestBase(Func<RouteValues, RouteValues> pathSelector)
 		[JsonIgnore]
 		TParameters IRequest<TParameters>.RequestParameters { get; set; } = new TParameters();
 
-
 		protected TOut Q<TOut>(string name) => RequestState.RequestParameters.GetQueryStringValue<TOut>(name);
 
 		protected void Q(string name, object value) => RequestState.RequestParameters.AddQueryStringValue(name, value);
+
 	}
 
+	public abstract class PlainRequestBase<TParameters> : RequestBase<TParameters>
```

Comment:
Ah, makes sense. :+1:  I knew there had to be a good reason, but it wasn't obvious just by viewing the diff.